neo4j-cli/aura/internal/subcommands/deployment/utils/response.go

```diff
@@ -0,0 +1,5 @@
+package utils
+
+func IsSuccessful(statusCode int) bool {
```

Comment:
This function already exists in the file `api/api.go` where `MakeRequest` is.

Currently it is not exposed, but I think it makes sense to make it public and use that instead of duplicating the logic as a separate util. Thoughts?